packages/beacon-node/src/eth1/eth1DataCache.ts

```diff
@@ -21,6 +21,6 @@ export class Eth1DataCache {
 
   async getHighestCachedBlockNumber(): Promise<number | null> {
     const highestEth1Data = await this.db.eth1Data.lastValue();
-    return highestEth1Data && highestEth1Data.blockNumber;
+    return highestEth1Data?.blockNumber || null;
```

Comment:
```suggestion
    return highestEth1Data?.blockNumber ?? null;
```

And a few other cases in this PR